props/saber_fett263_buttons.h

```diff
@@ -293,248 +664,3398 @@ class SaberFett263Buttons : public virtual PropBase {
+            SayNumber(calc_, SAY_WHOLE);
+            break;
+          case TWIST_IGNITION:
+            menu_type_ = MENU_TWISTON;
+            if (saved_gesture_control.twiston) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case THRUST_IGNITION:
+            menu_type_ = MENU_THRUSTON;
+            if (saved_gesture_control.thruston) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case STAB_IGNITION:
+            menu_type_ = MENU_STABON;
+            if (saved_gesture_control.stabon) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case FORCE_PUSH:
+            menu_type_ = MENU_FORCEPUSH;               
+            if (saved_gesture_control.forcepush) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case FORCE_PUSH_LENGTH:
+            menu_type_ = MENU_FORCEPUSH_LENGTH;
+            PlayMenuSound("mselect.wav");
+            calc_ = saved_gesture_control.forcepushlen;
+            SayNumber(calc_, SAY_WHOLE);
+            break;
+          case TWIST_RETRACTION:
+            menu_type_ = MENU_TWISTOFF;
+            if (saved_gesture_control.twistoff) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case LOCKUP_DELAY:
+            menu_type_ = MENU_LOCKUP_DELAY;
+            PlayMenuSound("mselect.wav");
+            calc_ = saved_gesture_control.lockupdelay;
+            SayNumber(calc_, SAY_MILLIS);             
+            break;
+          case CLASH_DETECTION:
+            menu_type_ = MENU_CLASH_DETECT;
+            PlayMenuSound("mselect.wav");
+            calc_ = saved_gesture_control.clashdetect;
+            SayNumber(calc_, SAY_WHOLE);
+            break;
+          case POWER_LOCK:
+            menu_type_ = MENU_POWERLOCK;
+            if (saved_gesture_control.powerlock) {
+              PlayMenuSound("mtrue.wav");
+              choice_ = true;
+            } else {
+              PlayMenuSound("mfalse.wav");
+              choice_ = false;
+            }
+            break;
+          case MAX_CLASH:
+            menu_type_ = MENU_MAX_CLASH;
+            PlayMenuSound("mselect.wav");
+            calc_ = saved_gesture_control.maxclash;
```

Comment:
calc_ doesn't seem like a good name for this variable.
Maybe just something like "integer_menu_value_" ?

Reply:
Can this be something you just "search and replace" after check in?  There's 83 instances.  If I do find and replace in a text editor it's easy but on github.com I have to do manually.

Reply:
So why edit code on github?
That seems tedious.
Maybe time to read up on how to edit code locally in a git branch?